### PR TITLE
matrix node docs

### PR DESCRIPTION
**Pull request by Jin1L**

### CODE REVIEW

File: docs/nodes/GENERATORS/SIMULATIONS/MATRIX/MATRIX.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+[//]: # (Examples)
+
+## Examples
+
+import Example1 from './examples/EX1/example.md';
+import App1 from '!!raw-loader!./examples/EX1/app.txt';
+import Data1 from '!!raw-loader!./examples/EX1/output.txt';
+
+<AppDisplay 
+  GLink='GENERATORS/SIMULATIONS/MATRIX'
```
Comment (smahmed776) [inline docs/nodes/GENERATORS/SIMULATIONS/MATRIX/MATRIX.md:34]: This prop is only given when there is no example available to help users to navigate to the node function file.

Comment (smahmed776) [reply docs/nodes/GENERATORS/SIMULATIONS/MATRIX/MATRIX.md:34]: `data` props must be provided for `<AppDisplay>` when an example is available

**Review by smahmed776 — CHANGES_REQUESTED** [review]
Left a couple of comments

**Comment by scorinaldi** [comment]
Ok @smahmed776 , looks like @Jin1L made the changes. Could you take another look at this one?

**Pull request closed without merging**